gke/README.md

```diff
@@ -77,23 +78,32 @@ Deploy a [containerized version of the Datadog Agent][7] on your Kubernetes clus
       --set clusterAgent.metricsProvider.enabled=true \
       --set providers.gke.autopilot=true \
       --set datadog.logs.enabled=true \
-      --set datadog.apm.enabled=true \
+      --set datadog.apm.portEnabled=true \
       datadog/datadog
   ```
 
-  See the [Datadog helm-charts repository][9] for a full list of configurable values.
+  See the [Datadog `helm-charts` repository][101] for a full list of configurable values.
 
 #### Admission Controller
  
-To use [Admission Controller](https://docs.datadoghq.com/containers/cluster_agent/admission_controller/?tab=operator) with Autopilot, set the [`configMode`](https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L922) of the Admission Controller to either `service` or `hostip`. 
+To use [Admission Controller][102] with Autopilot, set the [`configMode`][103] of the Admission Controller to either `service` or `hostip`. 
 
 Because Autopilot does not allow `socket` mode, Datadog recommends using `service` (with `hostip` as a fallback) to provide a more robust layer of abstraction for the controller. 
+
+[101]: https://github.com/DataDog/helm-charts/tree/master/charts/datadog#values
+[102]: https://docs.datadoghq.com/containers/cluster_agent/admission_controller/?tab=operator
+[103]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L922
```

Comment:
```suggestion
[103]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L1046
```

Reply:
Looks like this is the new location of the `configMode` parameter, but it might be worthwhile to confirm with the team whether there's a more stable link that can be provided